doc/internal/release_process.rst

```diff
@@ -5,37 +5,57 @@ This process is only meant for the project administrators, not users and develop
+4. Build the PyPI source distribution using ``python setup.py sdist build``.
 
-9. Once both builds have passed, make a pull request with the release branch to be merged into ``master`` and request code review.
+5. Upload the source distribution to TestPyPI  using ``twine upload --repository testpypi dist/*``. You will need to have the ``twine`` package installed and set up your ``$HOME/.pypirc`` correctly. See details `here <https://packaging.python.org/guides/using-testpypi/>`__.
 
-10. Once the build for the PR passes and the reviewers approve, merge the release branch into ``master``.
+6. Install the TestPyPI package as follows::
 
-11. Make sure that the ReadTheDocs build for ``master`` passes.
+    pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple rsmtool
 
-12. Tag the latest commit in ``master`` with the appropriate release tag and publish the release on GitHub.
+7. Then run some tests from a RSMTool working copy. If the TestPyPI package works, then move on to the next step. If it doesn't, figure out why and rebuild and re-upload the package.
 
-13. Make another PR to merge ``master`` branch into ``stable`` so that the ``stable`` ReadTheDocs build always points to the latest release.
+8. Build the new generic conda package by running the following command in the ``conda-recipe`` directory::
 
-14. Update the CI plan for RSMExtra (only needed for ETS users) to use this newly built RSMTool conda package. Do any other requisite changes for RSMExtra. Once everything is done, do a release of RSMExtra.
+    conda build -c conda-forge -c ets .
 
-15. Update the RSMTool conda environment on the ETS linux servers with the latest packages for both RSMTool and RSMExtra.
+9. Upload the package to anaconda.org using ``anaconda upload --user ets <package tarball>``. You will need to have the appropriate permissions for the ``ets`` organization. 
 
-16. Send an email around at ETS announcing the release and the changes.
+10. Create pull requests on the `rsmtool-conda-tester <https://github.com/EducationalTestingService/rsmtool-conda-tester/>`_ and `rsmtool-pip-tester <https://github.com/EducationalTestingService/rsmtool-pip-tester/>`_ repositories to test the conda and TestPyPI packages on Linux and Windows.
 
-17. Create a `Dash <https://kapeli.com/dash>`_ docset from the documentation by following the instructions :ref:`here <dash_docset>`.
+11. Draft a release on GitHub while the Linux and Windows package tester builds are running.
+
+12. Once both builds have passed, make a pull request with the release branch to be merged into ``master`` and request code review.
+
+13. Once the build for the PR passes and the reviewers approve, merge the release branch into ``master``.
+
+14. Upload source and wheel packages to PyPI using ``python setup.py sdist upload`` and ``python setup.py bdist_wheel upload``
+
+15. Make sure that the ReadTheDocs build for ``master`` passes.
```

Comment:
Add how to do this?

Reply:
Sure, I'll add the URL to the badge which will say whether the build failed or not.